library/src/scala/tasty/Reflection.scala

```diff
@@ -51,10 +51,9 @@ trait Reflection extends reflect.Types { reflectSelf: CompilerInterface =>
 
   // ----- Tree -----------------------------------------------------
 
-  given TreeOps as Tree.type = Tree
+  object Tree
```

Comment:
It seems the object is not used?

Reply:
Indeed. I kept it because as a placeholder in case we need to add methods to it. We had a case in the past where we had to add one of those and broke some code because the imports behaved differently. It might also be useful for documentation purposes.